modules/navier_stokes/doc/content/source/fvkernels/INSFVMomentumAdvection.md

```diff
@@ -9,6 +9,33 @@ case is the momentum component $\rho u_i$ where $u_i$ denotes the x, y, or z
 component of the velocity in Cartesian coordinates, or the r or z component of
 the velocity in RZ coordinates.
 
+The default is to use the same $a$ coefficients when computing the kernel residuals
+that strongly affect the solution and when computing the Rhie-Chow
+velocity which weakly affects the solution by perturbing the linearly
+interpolated velocity as shown in the Rhie-Chow velocity computation shown in
+[INSFVRhieChowInterpolator.md], which we reproduce here:
+
+\begin{equation}
+\label{rcvel}
+\bm{v}_f = \overline{\bm{v}_f} - \overline{\bm{D}_f}\left(\nabla p_f - \overline{\nabla p_f}\right)
+\end{equation}
```

Comment:
use a_s here or write a second line with the as.
you refer to as not Ds in the next paragraph